tests/test_hazard.py

```diff
@@ -77,8 +77,8 @@ def test_non_parametric_gm_prob():
     im_values = pd.Series(index=index, data=values)
     result = sha.hazard.non_parametric_gm_excd_prob(im_level, im_values)
 
-    assert result.loc["rupture_1"] == 0.5
-    assert result.loc["rupture_2"] == 2 / 3
+    assert np.isclose(result.loc["rupture_1", 2], 0.5)
+    assert np.isclose(result.loc["rupture_2", 2], 2 / 3)
```

Comment:
```suggestion
    assert result.loc["rupture_1", 2] == pytest.approx(0.5)
    assert result.loc["rupture_2", 2] == pytest.approx(2 / 3)
```
Gives better output when tests fail.